src/AI/agents/scripts/make_injection_fixture.py

```
@@ -0,0 +1,111 @@
+    for offset in offsets:
+        out += f"{offset:010d} 00000 n \n".encode()
+    out += (
+        f"trailer\n<< /Size {len(objects) + 1} /Root 1 0 R >>\n"
+        f"startxref\n{xref_at}\n%%EOF\n"
+    ).encode()
+    return bytes(out)
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("target", nargs="?", default="/tmp/hostile-skjema.pdf")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/hostile-skjema.pdf"

(S108)